app/models/sample.rb

```diff
@@ -470,10 +477,31 @@ def can_be_included_in_submission?
     end
   end
 
+  def control_formatted
+    return nil if control.nil?
+
+    return 'No' if control == false
+
+    case control_type
+    when 'positive'
+      'Yes (positive)'
+    when 'negative'
+      'Yes (negative)'
+    else
+      'Yes'
+    end
+  end
+
```

Comment:
Would
```
  def control_formatted
    return nil if control.nil?

    return 'No' if control == false
  "Yes (#{control_type||'Unspecified'})"
  end
```

be better? Its not quite the same behaviour, and perhaps I'm a little obsessed with removing branches.